src/app/user.ts

```diff
@@ -6,4 +6,11 @@ export interface User {
     lastName: string,
```

Comment:
User ID field needs to be updated to reflect that we're storing a UUID and not a number